vendor/vendor.json

```diff
@@ -94,6 +94,12 @@
 			"tree": true
 		},
 		{
+			"checksumSHA1": "nbCxiVT48CWEg6l8L4SEJ5VfN9c=",
+			"path": "github.com/golang/groupcache/lru",
```

Comment:
Vendoring this in since the existing vendored `lru` package doesn't support uncapped LRUs, which is needed for our own bytes-counting based cache.